portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -34,10 +34,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     names.add("leah");
 
     response.setContentType("text/html;");
-
-    for(int i = 0; i < names.size(); i++){
-        response.getWriter().println(names.get(i));
+    for(int i = 0; i < names.size(); i ++){
```

Comment:
So an arraylist of names in Json would look like this:
`{ "names" : ["many", "zoe", "leah"]
}`
You can use the library **gson** to convert a java object (like the ArrayList of names) to JSON like so:
```
import com.google.gson.Gson;
Gson gson = new Gson();
String json = gson.toJson(names);
```

Reply:
+1